packages/client-search/src/types/SaveDictionaryEntriesOptions.ts

```diff
@@ -0,0 +1,8 @@
+import { DictionaryEntry } from './DictionaryEntry';
+
+export type SaveDictionaryEntriesOptions = {
```

Comment:
I think this should be called something like 

```suggestion
export type DictionaryEntriesOptions = {
```

so that it doesn't have the `save` part in it, it's used in more params

Reply:
Definitely!